### PR TITLE
Prepare release of SDK 1.6.1

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
FYI, the diff for translations is quite large, in particular for rule_en.xml.

### CODE REVIEW

**Review by pdonohoe — APPROVED** [review]
There are no fields used in the EFX templates in prepare/1.6.1 that are not defined in fields.json in release/1.3.4 :)

**Comment by bertrand-lorentz** [comment]
View templates were all successfully compiled manually. This confirms their syntax is valid and they do not reference any node or field that does not exist in this version.

File: translations/auxiliary_ga.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">
 <properties>
-<comment>auxiliary in Galician. File generated from metadata database version 1.6.0 created on the 2023-02-17T12:00</comment>
+<comment>auxiliary in Irish. File generated from metadata database.</comment>
```
Comment (rousso) [inline translations/auxiliary_ga.xml:4]: The filename has _ga which I assume stands for Galician. The comment was changed from Galician to Irish. The actual labels are in Irish. Something is wrong with this file.

File: translations/business-term_ga.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">
 <properties>
-<comment>business-term in Galician. File generated from metadata database version 1.6.0 created on the 2023-02-17T12:00</comment>
+<comment>business-term in Irish. File generated from metadata database.</comment>
```
Comment (rousso) [inline translations/business-term_ga.xml:4]: Same here. Is the filename correct?

File: translations/view_ga.xml
Comment (rousso) [inline translations/view_ga.xml:1]: Check if the filename is correct. "ga" does not seem to be a two letter code for Irish to me.

File: translations/rule_ga.xml
Comment (rousso) [inline translations/rule_ga.xml:1]: Check if the filename is correct. "ga" does not seem to be a two letter code for Irish to me.

File: translations/notice_ga.xml
Comment (rousso) [inline translations/notice_ga.xml:1]: Check if the filename is correct. "ga" does not seem to be a two letter code for Irish to me.

File: translations/group_ga.xml
Comment (rousso) [inline translations/group_ga.xml:1]: Check if the filename is correct. "ga" does not seem to be a two letter code for Irish to me.

File: translations/field_ga.xml
Comment (rousso) [inline translations/field_ga.xml:1]: Check if the filename is correct. "ga" does not seem to be a two letter code for Irish to me.

File: translations/codelist_ga.xml
Comment (rousso) [inline translations/codelist_ga.xml:1]: Check if the filename is correct. "ga" does not seem to be a two letter code for Irish to me.

**Review by rousso — CHANGES_REQUESTED** [review]
Apart from the *_ga.xml files in translations where the filename seems wrong, everything else looks good to me. We should check the filename though because the content of those files is in Irish so I would expect to see *_ie.xml

**Comment by bertrand-lorentz** [comment]
> Apart from the *_ga.xml files in translations where the filename seems wrong, everything else looks good to me. We should check the filename though because the content of those files is in Irish so I would expect to see *_ie.xml

The 2 letter code for Irish is "ga". So no problem here.
So as agreed with Yannis I'm merging this.
Note: checks are failing because they are not target for this version of the SDK, which causes false alerts.

**Pull request merged**